contents/blog/best-matomo-alternatives.mdx

```diff
@@ -0,0 +1,462 @@
+
+### How does Amplitude compare to Matomo?
+
+<ComparisonTable column1="Amplitude" column2="Matomo">
+  <ComparisonRow column1={true} column2={false} feature="Free tier" description="Free forever option, no credit card required" />
+  <ComparisonRow column1={true} column2={true} feature="Web analytics" description="Pre-built dashboard of aggregate traffic, sources, campaigns, and session metrics" />
+  <ComparisonRow column1={true} column2={true} feature="Product analytics" description="Custom events, trends, funnels, paths, and retention" />
+  <ComparisonRow column1={false} column2={false} feature="Autocapture" description="Capture clicks and pageviews without manual instrumentation" />
+  <ComparisonRow column1={true} column2={true} feature="Cohort analysis" description="Combine users into cohorts for analysis" />
+  <ComparisonRow column1={false} column2={true} feature="Search tools" description="Get keyword data and track search ad performance" />
+  <ComparisonRow column1={true} column2={true} feature="Session replay" description="Watch real users on your site; discover friction points" />
+  <ComparisonRow column1={false} column2={true} feature="Heatmaps" description="Visualize where users click and interact on your site" />
+  <ComparisonRow column1={true} column2={true} feature="A/B testing" description="Run multivariate tests and see the impact of changes with custom goals and reports" />
+  <ComparisonRow column1={false} column2={true} feature="Open source" description="Audit code, contribute to roadmap, and build integrations" />
+  <ComparisonRow column1={true} column2={true} feature="EU hosting" description="Store your data in the EU for compliance" />
+</ComparisonTable>
+
+### How popular is Amplitude?
+
+Amplitude and Matomo are neck and neck in terms of popularity. According to [BuiltWith](https://trends.builtwith.com/analytics/Amplitude), as of October 2024, 19,098 of the top 1 million websites deploy Amplitude and 20,816 use Matomo.
+
+### Why do companies use Amplitude?
+
+According to G2 reviews, Amplitude users appreciate three key aspects:
+
+1. **Simple to use:** Amplitude makes it easy for non-technical users to get insights about their product and make improvements. Amplitude is built for users like product managers and marketers, making it a popular choice for them.
+
+2. **Built-in A/B testing:** Amplitude offers integrated experimentation features. This enables companies to run experiments on existing cohorts, and then analyze the data in a single place.
+
+3. **Become data-driven:** Amplitude users appreciate it helps them become data-driven. It becomes easy to add data, visualize it, and make decisions. It makes data accessible to them.
+
+> #### Bottom line
+> Amplitude is a popular choice for product teams looking for simple product analytics. Since it offers a similarly wide range of features as Matomo, it makes for a solid choice.
+
+## 7. Pendo
```

Comment:
I'm not sure Pendo really belongs in this list. It's a long way from what Matomo is as a tool, so not a viable alternative imo.